.github/workflows/template-validation.yaml

```diff
@@ -4,13 +4,9 @@
 name: Template Validation
 
 on:
-  #push:
-  #  branches: [ $default-branch ]
   pull_request:
-    branches: [ master ]
-    # don't require the build on docs only changes?
-    #  paths-ignore:
-    #  - 'Documentation/**'
+  push:
```

Comment:
does this mean validation is only run on a push? the pull_request field is empty

Reply:
https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows nvm ,from the docs here it looks like this setting is correct

Reply:
yeah, i was confused too.  i think it is the yaml syntax, like "on push i have no extra settings, on pull request i do since they're embedded underneath.  without any settings i think you just do `on: [push, pullrequest]`